assets/stylesheets/screens/not_found.scss

```
@@ -0,0 +1,19 @@
+.not-found {
```

Missing the body tag here:

```suggestion
body.not-found {
```

---

Fixed in 3853dc031eddc5caad6b6194e9887f20a3332492